client/coral-admin/src/components/UserDetailComment.js

```diff
@@ -78,9 +78,11 @@ class UserDetailComment extends React.Component {
           <CommentAnimatedEdit body={comment.body}>
             <div className={styles.bodyContainer}>
               <div className={styles.body}>
-                <CommentBodyHighlighter
-                  suspectWords={suspect}
-                  bannedWords={banned}
+                <CommentFormatter
+                  settings={{
+                    suspectWords: suspect,
+                    bannedWords: banned,
+                  }}
                   body={comment.body}
```

Comment:
I'd advise using shallow properties instead of the nested object `settings`.

``js
<CommentFormatter
  suspectWords={suspect}
  bannedWords={banned}
  body={comment.body}
``

This works better for optimisations.